### DISCUSSION TITLE
[SOLVED] How to sync read status with Nextcloud?

### DISCUSSION THREAD

**Discussion by wpost** [martinrotter/rssguard#761] [Q&A] [answered]
I have RSS Guard 3.9.2 on openSUSE 15.3 logged in to my Nextcloud account. All works well except that when I mark a message as read in Nextcloud News, it continues to appear in RSS Guard as unread. To reproduce:

1. In RSS Guard, add a Nextcloud account. In a web browser, have Nextcloud News open. You should see the same messages in both RSS Guard and Nextcloud News, and least one message should be unread.
2. In Nextcloud News, mark a message as read. 
3. In RSS Guard, Select Accounts - (account name) - Synchronize Folders and Other Items. Wait a few seconds.

Expected result: In RSS Guard, the message is either marked as read or disappears from view, depending upon whether RSS Guard is configured to show read messages.

Observed result: No change in RSS Guard. There, the message remains unread.

The other way around does work: messages I mark as read in RSS Guard do quickly appear as read in Nextcloud News.

My guess is that I am doing something wrong and/or have something misconfigured in RSS Guard. Where might I start looking to find my error?

**Comment by martinrotter** [comment, marked as answer]
Hi!

RSS Guard supports two-way fully automatic sync with Nextcloud News. It seems that in your case some is very wrong. Note that synchronization in direction RSS Guard --> Nextcloud is done automatically once a minute.

The sync in other direction is done when you "Fetch" the given feed. Either via "Fetch all" button or individual feed with "Fetch selected" button.

Button "Folders and Other Items" is there for another purpose - it synchronizes THE LIST of feeds/categories itself. It does not fetch new/modified article.

Also, your version of RSS Guard is **VERY OUTDATED**. UPDATE TO LATEST VERSION!!!!!!

Also, post here how your RSS Guard Nextcloud settings dialog looks like. Maybe you have "Fetch unread articles only" option enabled?

What version of Nextcloud you use? I will test.

**Reply by wpost** [reply]
Thank you! You were correct: disabling "download unread articles only" in the account settings got sync working as expected.

Regarding my version, that was silly of me. The first thing I should have confirmed, and certainly before asking for help, was that I was running latest stable. I have upgraded and thank you for the reminder. That wasn't related to my trouble but running latest stable is always a good idea.

Many thanks for my new favorite feed reader.

**Reply by martinrotter** [reply]
No problem.

I understand that there is a bit of misunderstanding regarding "fetch unread articles only" feature. Naturally when checked you would want fetch UNREAD articles + you would want to receive state changes for articles which are already unread in your RSS Guard but were marked READ on the server by some other mean (in web interface, other clients etc). This is sadly hard to do with Nextcloud News API.
